modules/special-rules/pages/psychic-beacon.adoc

```diff
@@ -8,10 +8,12 @@ Genestealer Patriarchs emit a strong psychic signal, which guides the parent Hiv
 
 When you field an army that includes detachments that have the {psychic-beacon} ability:
 
-* You can deploy any Mycetic Swarms that you have in the first turn, rather than roll for their arrival as Reserves.
-+
-{blank}[TODO: Is this the right interpretation? Seems like it might be overpowered -- include a single small Genestealer Cult detachment, and all Mycetic Swarms in a Tyranid army get to deploy on turn 1, no roll for Reserves required...]
+* For each undestroyed detachment with the {psychic-beacon} ability, you may re-roll one failed Reserves roll per turn.
+
+[NOTE.e40k]
+====
+The _Epic 40,000_ version of this ability was exceptionally cheap and powerful, enabling all Mycetic Swarm detachments in the army to automatically arrive in turn 1. This revised version scales with the number of detachments with the ability.
```

Comment:
A bit of the usual editing, but also see my query after the suggestion:

```suggestion
The _Epic 40,000_ version of this ability is exceptionally cheap and powerful — for only a small cost, it allows all Mycetic Swarm detachments in the army to arrive in turn 1, automatically.
We've revised the ability so that it scales with the number of detachments that have the ability, and so that it improves rather than bypasses the odds on the Reserve Detachment Arrival table.
```

This new version of the ability seems much better but: Aside from the 'non-scaling' and 'automatic' problems, the original was also exceptional in that it allowed Mycetic Swarms to arrive on the very **first** turn — whilst players can't start to roll for Reserves until the **second** turn. Did you intend that this ability should no longer make it possible **at all** for Mycetic Swarms to arrive on the very first turn? (If yes, then great, that's obviously the simplest and most consistent — but it is another big difference so I thought I'd check.)